wiki/Tournaments/GTS/CGTS_2019/en.md

```diff
@@ -0,0 +1,199 @@
+---
+tags:
+- CGTS 2019
+- CGTS2019
+---
+
+# Continental Global Taiko Showdown 2019
+
+![CGTS 2019 Logo](img/logo.png)
+
+The **Continental Global Taiko Showdown 2019** (***CGTS 2019***) is an advanced worldwide double-elimination osu!taiko tournament hosted by ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263) and ![][flag_FR] [TLQ\_Yoshii](https://osu.ppy.sh/users/7157133). It is the first installment of the Continental Global Taiko Showdown and part of the Global Taiko Showdown series.
```

Comment:
what does "advanced" mean? is it referring to a rank restriction? if so just write that instead

Reply:
Seems i forgot to delete this part, copied from agts xd